dockers/README.md

```diff
@@ -0,0 +1,91 @@
+# How to locally test
+First build the funcX web base image:
+```bash
+cd dockers/base
+docker build -t funcx-web-base .
+```
+Next, we need to ensure that we have the correct directory structure.  You will need to move files around so that your
+tree has this:
+```
+funcx-faas/
+    docker-compose.yml
+    funcX/
+    funcx-web-service/
+```
+It is important that the main `docker-compose.yml` file and the two repos are at the same level. (This is due to the
+way in which the build context is sent to the docker daemon, and other solutions are clunky.)
+## Secrets
+#### **WARNING**: make sure to have the `dockers/secrets` directory in your `.gitignore` if it is not already.  
+You'll need a few secrets for the app to work properly.  First, let's do the Globus secrets: the funcX Globus Confidential Client ID and Key.
+Place them in `dockers/secrets/globus_client.txt` and `dockers/secrets/globus_key.txt`.  Next, we need to create a 
+self-signed cert/key pair, also in the `secrets` directory.  This will be for our web service, which will have the
+domain `funcx.org` within the Docker network.   
+```
```

Comment:
this should be in `funcx-web-service/dockers/secrets` -- probably do a `cd` here